Src/Sarif.PartnerMatcher.Function/SpamAnalyzer.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Function
+{
+    public class SpamAnalyzer
```

Comment:
>public class [](start = 4, length = 12)

make internal static class. We should use as much internal as we can, since we don't want to expose things that aren't needed.

Reply:
done